uri/exid-types/RFN.yaml

```diff
@@ -1,13 +1,15 @@
 %YAML 1.2
 ---
+label: RFN
+
 lang: en
 
 type: uri
 
 uri: https://gedcom.io/terms/v7/RFN
 
 specification:
-  - Record File Number
+  - Permanent Record ID
```

Comment:
Here the two files had different values.  Perhaps "Record File Number" should be moved to the label?